package.json

```diff
@@ -27,14 +27,15 @@
     "/dist",
     "/schemas"
   ],
-  "main": "dist/enigma.js",
+  "main": "src/index.js",
```

Comment:
This would make sense if the source code was written using the CommonJS module format. ES2015 imports / exports is not (yet) supported in NodeJS.